test/extended/networking/egress_firewall.go

```diff
@@ -100,6 +126,44 @@ func doEgressFwTest(f *e2e.Framework, oc *exutil.CLI, manifest string, nodeSelec
 
 	o.Expect(sendEgressFwTraffic(f, oc, egressFWTestPod, nodeSelectorSupport)).To(o.Succeed())
 
+	// TODO:
+	// - Check for coredns-ocp-dnsnameresolver plugin.
+	// - Remove TecHPreview check when feature is GA.
+	if networkPluginName() == OVNKubernetesPluginName && exutil.IsTechPreviewNoUpgrade(oc) {
+		cfg, err := e2e.LoadConfig()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		dc, err := dynamic.NewForConfig(cfg)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		dnsNameResolverClient := dc.Resource(schema.GroupVersionResource{Group: "network.openshift.io", Resource: "dnsnameresolvers", Version: "v1alpha1"})
+
+		e2e.Logf("fetching DNSNameResolver objects list")
+		obj, err := dnsNameResolverClient.List(context.Background(), metav1.ListOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		dnsNameResolvers := objx.Map(obj.UnstructuredContent())
+		items := objects(dnsNameResolvers.Get("items"))
+		o.Expect(len(items)).ShouldNot(o.Equal(0))
+
+		for _, item := range items {
+			dnsName := item.Get("spec.name").String()
+			resolvedNames := item.Get("status.resolvedNames").InterSlice()
+			o.Expect(len(resolvedNames)).Should(o.Equal(1))
+			name, ok := resolvedNames[0].(nwv1alpha1.DNSNameResolverResolvedName)
+			o.Expect(ok).Should(o.Equal(true))
+			switch dnsName {
+			case "docs.openshift.com":
+				o.Expect(name.DNSName).Should(o.Equal("docs.openshift.com"))
```

Comment:
```suggestion
				o.Expect(name.DNSName).Should(o.Equal("docs.openshift.com."))
```
DNSNameResolver objects stores the DNS names with a trailing period.